packages/web-components/test/display/natValue/stringifyRatio.test.js

```diff
@@ -31,7 +31,6 @@ describe('stringifyRatio', () => {
     const ethPrice = harden(
       makeRatio(
         158724n, // value of 1 eth in cents
-        // @ts-expect-error fake brand
```

Comment:
Does yarn 4 have an issue with `@ts-expect-error`?

Reply:
I ran into something in merging or rebasing and after having dealt with it several times in in the .ts PR I wanted a quieter solution